docs/B-Computations/testing-and-debugging.md

```diff
@@ -241,11 +245,11 @@ To trace through execution of our program using the built-in debugger
 * Observe the values under the Locals tab in the Window below the source code
 * Press F10 until the input prompt appears and answer the prompt by entering 3
 * Observe the values under the Locals tab in the window below the source code
-* Press F5, note the position of the arrow identifying the next statement to be executed and observe the value of total
+* Press F5, note the position of the arrow identifying the next statement to be executed, and observe the value of total
 * Press F5 and answer the prompt by entering a value of 2
-* Press F5, note the position of the arrow identifying the next statement to be executed and observe the value of total
+* Press F5, note the position of the arrow identifying the next statement to be executed, and observe the value of total
 * Press F5 and answer the prompt by entering a value of 0
-* Press F5, note the position of the arrow identifying the next statement to be executed and observe the value of total
+* Press F5, note the position of the arrow identifying the next statement to be executed, and observe the value of total
 * Press F5 and
```

Comment:
![image](https://user-images.githubusercontent.com/44437742/142316169-0bd4ffac-c89b-4fd1-96f5-bf139f6e4ea5.png)
1) Add a white space at the beginning of line 256
2) I wonder that is it appropriate to make line 254 and 255 become sub bullet points of line 253 like this:

* Press F5 and
  * Select the command prompt window and observe the program output
  * Select the source code window